tests/test_distributed_scheduler.py

```
@@ -331,3 +331,84 @@ async def task():
+
+    scheduler = DistributedScheduler(communicator, concurrency=2, metrics_interval=0.1)
+
+    async def fake_get_peer_loads() -> dict[str, float]:
+        return {
+            "http://peer1/api/v1/peer/message": 0.1,
+            "http://peer2/api/v1/peer/message": 0.25,
+            "http://peer3/api/v1/peer/message": 0.4,
+        }
+
+    def fake_get_peer_telemetry_map(
+        max_age: float = 0.0, include_self: bool = False
```

Ruff (0.13.2)

Unused function argument: `max_age`

(ARG001)